packages/react-storybook/config/api-extractor.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
If you want it's probably okay to get rid of API Extractor for this package

Reply:
Im gonna keep it for consistency reasons. in general every package should use common tooling in terms of rolluping types and producing docs.